docs/convert.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[convert-to-ecs]]
+== Converting an implementation to ECS
+
+A common schema helps you correlate and use data from various sources. 
+
+Fields for most Elastic modules and solutions (version 7.1 and later) are mapped
+to the Elastic Common Schema. You may want to map data from other
+implementations to ECS to help you correlate data across all of your products
+and solutions.
+
+Before you start a conversion, be sure that you understand the basic <link here>.
```

Comment:
Let's keep a todo to finish this sentence. Or can we for now just say "the basics outlined below:"?